test/decoders/test_video_decoder.py

```diff
@@ -11,7 +11,9 @@
 import torch
 from torchcodec import FrameBatch
 
-from torchcodec.decoders import _core, VideoDecoder
+from torchcodec.decoders import _core, VideoDecoder, VideoStreamMetadata
+from torchcodec.decoders._audio_decoder import AudioDecoder
```

Comment:
Since we're testing both video and audio in the same file, we should probably change this file name to `test_decoders.py` or something similar.